### PR TITLE
use variables | Added PowerShell conceptual tab

### PR DESCRIPTION
**Pull request by v-abdriaz**
Added PowerShell commands and tabs.

### CODE REVIEW

**Comment by v-abdriaz** [comment]
#sign-off

File: docs-ref-conceptual/azure-cli-variables.md [hunk 1]
```diff
@@ -44,6 +47,22 @@ osType=$(az vm get-instance-view --resource-group $MyResourceGroup \
...
+
+```azurecli
+# PowerShell script
+# Assign values to variables
+$MyResourceGroup = "ContosoRGforVM"
+$MySubscription = "Contoso subscription"
+$vmName = "VM01"
+
+# Get a value for a variable based on an existing virtual machine
+$osType = (Get-AzVM -ResourceGroupName $MyResourceGroup -Name $vmName -SubscriptionName $MySubscription).StorageProfile.OsDisk.OsType
```
Comment (dbradish-microsoft) [inline docs-ref-conceptual/azure-cli-variables.md:60]:
```suggestion
$osType=$(az vm get-instance-view --resource-group $MyResourceGroup `
   --name $vmName --subscription $MySubscription `
   --query 'storageProfile.osDisk.osType' --output tsv)
```

Comment (dbradish-microsoft) [reply docs-ref-conceptual/azure-cli-variables.md:60]: @v-abdriaz , This is an Azure CLI article and ```azurecli code block that should only contain Azure CLI commands.

File: docs-ref-conceptual/azure-cli-variables.md [hunk 2]
```diff
@@ -44,6 +47,22 @@ osType=$(az vm get-instance-view --resource-group $MyResourceGroup \
...
+
+# Create a disk of the same type by using the variable value
+New-AzDisk -ResourceGroupName $MyResourceGroup -DiskName DestinationDisk -DiskSizeGB 20 -OsType $osType
```
Comment (dbradish-microsoft) [inline docs-ref-conceptual/azure-cli-variables.md:63]:
```suggestion
az disk create --resource-group $MyResourceGroup --name DestinationDisk --size-gb 20 --os-type $osType
```

Comment (dbradish-microsoft) [reply docs-ref-conceptual/azure-cli-variables.md:63]: @v-abdriaz , The point of this update is to show the customer how to run Azure CLI commands in a PowerShell environment. We should not be replacing Azure CLI commands with Azure PowerShell commands.  We need to make the Azure CLI command work.

**Review by dbradish-microsoft — CHANGES_REQUESTED** [review]

File: docs-ref-conceptual/azure-cli-variables.md [hunk 3]
```diff
@@ -44,6 +47,22 @@ osType=$(az vm get-instance-view --resource-group $MyResourceGroup \
 # Create a disk of the same type by using the variable value
 az disk create --resource-group $MyResourceGroup --name DestinationDisk --size-gb 20 --os-type $osType
 ```
+# [PowerShell](#tab/powershell)
+
+```azurecli
```
Comment (dbradish-microsoft) [inline docs-ref-conceptual/azure-cli-variables.md:52]:
```suggestion
```azurecli-interactive
```

**Comment by dbradish-microsoft** [comment]
@v-abdriaz , I'm a bit confused.  Where did the PowerShell conceptual tab go?  I gave you the syntax in one of the conversations.

**Pull request merged**